src/main/java/uk/gov/hmcts/ccd/data/userprofile/UserProfileRepository.java

```diff
@@ -64,10 +69,14 @@ public UserProfile createUserProfile(UserProfile userProfile, final String actio
         userProfileAuditEntityRepository.createUserProfileAuditEntity(userProfile,
                                                                       CREATE,
                                                                       actionedBy,
-                                                                      userProfile.getWorkBasketDefaultJurisdiction());
+                                                                      getFirstJurisdiction(userProfile));
         return UserProfileMapper.entityToModel(userProfileEntity);
     }
 
+    private String getFirstJurisdiction(UserProfile userProfile) {
+        return userProfile.getJurisdictions().stream().findFirst().map(Jurisdiction::getId).orElse(null);
```

Comment:
for my knowledge, what's the rationale of this change?

Reply:
there is no WorkBasketDefaultJurisdiction so.. audit with first Jurisdiction.